trade-finance-manager-ui/server/controllers/case/cancellation/validation/validate-reason-for-cancelling.ts

```diff
@@ -0,0 +1,19 @@
+import { ErrorSummaryViewModel, ReasonForCancellingErrorsViewModel } from '../../../../types/view-models';
+
+const MAXIMUM_LENGTH = 1200;
```

Comment:
Is there `libs/common` constant for this, if not might be worth importing there as it will be quite common for current and future textareas.